roles/example-cnf-app/tasks/retry-trex.yaml

```diff
@@ -1,9 +1,7 @@
 ---
 - name: Set retry variables
   set_fact:
-    trex_app_cr_name: trex-app1
-    trex_app_retry_count: "{{ trex_app_retry_count | default(0) | int + 1 }}"
-    trex_app_retry_max: "{{ trex_app_retry_max | default(5) | int }}"
+    trex_app_cr_name: trex-app
```

Comment:
Can you leave the name as trex-app1 ? I used this name to remove the events that pollutes the results during the upgrade.

Reply:
Or I can update the upgrade hook after we merge this, as you wish :) (I agree, it is an hard-coded value)

Reply:
we discussed by chat, and we'll go with the hook upgrade. Thank you @capolrik !